docs/develop/basics/hackathon-guide.md

```diff
@@ -32,8 +152,12 @@ Smart Contracts are the back-end of your application, which lives on the blockch
   
 - We currently support developing smart contracts in
   - [Rust](https://www.rust-lang.org/)
-  - [AssemblyScript](https://assemblyscript.org/introduction.html)
+  - [AssemblyScript](https://assemblyscript.org/introduction.html) (Note: AssemblyScript is not recommended for mission-critical contracts)
+
+
+## Boilerplate apps
 
+TBD
```

Comment:
maybe we can put some demo projects here as well?

Reply:
totally! Working on updating near.dev as we speak which will have more open source example projects.

Reply:
I have added a table with links/descriptions to all the examples available at `near.dev`.

I can update it as soon as we have more projects available.